### PR TITLE
Document the steps needed before creating a release

### PR DESCRIPTION
**Pull request by andybalaam**

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -1,4 +1,19 @@
-# Steps for releasing `matrix-sdk-crypto-wasm`
+# Releasing `matrix-sdk-crypto-wasm`
+
+## Before you release
+
+Assuming you are making a release to get the latest Rust code, you should bump
+the version of `matrix-rust-sdk` we are depending on in `Cargo.toml`.
```
Comment (richvdh) [inline RELEASING.md:6]: it's `Cargo.lock` that needs the update, not `Cargo.toml`.

File: RELEASING.md [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+
+At time of writing, Cargo.toml has `git = "https://github.com/matrix-org/matrix-rust-sdk"`,
+which picks up the latest version on the default branch in Git. This means that
+we can update the version by following these steps:
+
+1. Ensure `.cargo/config` does not contain the `patch` section for local
+   development recommended in `README.md`.
```
Comment (richvdh) [inline RELEASING.md:13]:
```suggestion
1. Ensure `.cargo/config` does **not** contain the `patch` section for local
   development recommended in `README.md`.
```

File: RELEASING.md (hunk 1 above)
Comment (andybalaam) [reply RELEASING.md]: Updated.

**Review by richvdh — APPROVED** [review]
lgtm

**Pull request merged**